torch/csrc/jit/codegen/cuda/root_domain_map.cpp

```diff
@@ -936,6 +940,13 @@ void ComputeAtRootDomainMapBuilder::handle(TensorView* tv) {
       mapAllConsumers(DomainKey(td, id));
     }
   }
+  if (td->hasRFactor()) {
+    for (auto id : TensorDomain::noReductions(td->getRootDomain())) {
```

Comment:
Can you pull: https://github.com/csarofeen/pytorch/pull/1548/files#diff-296b2445f5a818c3240b26185b37b309fa554f56b909f73109917cbdfd580333R1375-R1390 into this PR?